client/src/pages/SignUp/SignUp.tsx

```diff
@@ -40,18 +41,26 @@ export default function Register(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} sm={12} md={12} elevation={6} component={Paper} square>
         <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
+          <AuthHeader asideText="BECOME A SITTER" />
+          <Box className={classes.signupOuterContainer} width="100%" maxWidth={600} p={3} alignSelf="center">
+            <Grid className={classes.signupContainer} container elevation={6} component={Paper}>
               <Grid item xs>
                 <Typography className={classes.welcome} component="h1" variant="h5">
-                  Create an account
+                  Sign up
                 </Typography>
               </Grid>
+              <SignUpForm handleSubmit={handleSubmit} />
+              <Box className={classes.extraSection}>
```

Comment:
The MUI `Box` component has flexbox built into it, you can pass props to it instead of using CSS classes, please see below:

Eg: `<Box display="flex" alignItems="center"></Box>`

Reply:
Applied